django_project/settings.py

```diff
@@ -140,3 +140,6 @@
 
 # Set max files in a request
 MAX_FILES_PER_REQUEST = 50
+
+#Minimal string length considered similarity
+SIMILARITY_LENGTH = 50
```

Comment:
I would suggest adding `MIN_` prefix for this variable.

Reply:
And add space between `#` character and first letter of the commentary

Reply:
done